src/Microsoft.Android.Runtime.NativeAOT/Android.Runtime.NativeAOT/JavaInteropRuntime.cs

```diff
@@ -33,6 +33,7 @@ static void JNI_OnUnload (IntPtr vm, IntPtr reserved)
 	[UnmanagedCallersOnly (EntryPoint="Java_net_dot_jni_nativeaot_JavaInteropRuntime_init")]
 	static void init (IntPtr jnienv, IntPtr klass)
 	{
+		JniTransition   transition  = default;
```

Comment:
Looks like this is a struct, and no need to check if it's "empty"? Seems like the methods are no-op:
* https://github.com/dotnet/java-interop/blob/main/src/Java.Interop/Java.Interop/JniTransition.cs

Reply:
Yes, methods are a no-op if the `struct` "default constructor" is used.  The non-default constructor can't be used until *after* a `JniRuntime` exists, which made structure "weird".